.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I think this should live in the main repo 😊

Reply:
We can store common templates here and override it in other repos

Reply:
looks better now!